src/test/java/test/ronin/muserver/sample/AppTest.java

```diff
@@ -0,0 +1,67 @@
+package test.ronin.muserver.sample;
+
+import io.netty.util.ResourceLeakDetector;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import ronin.muserver.MuServer;
+import ronin.muserver.sample.App;
+
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.SSLSocketFactory;
+import javax.net.ssl.TrustManager;
+import javax.net.ssl.X509TrustManager;
+import java.io.IOException;
+import java.security.KeyManagementException;
+import java.security.NoSuchAlgorithmException;
+
+public class AppTest {
+
+    private MuServer server;
+
+    @AfterEach void tearDown() {
+        server.stop();
+    }
+
+    @Test void hitTheSample() throws IOException, KeyManagementException, NoSuchAlgorithmException {
+        ResourceLeakDetector.setLevel(ResourceLeakDetector.Level.PARANOID);
+
+        server = App.mu_server_builder.start();
```

Comment:
Discovered a port clash here when I left the main() one running - which led me to the builder rabbit hole